lib/tests/e2e/suite.go

```diff
@@ -30,7 +30,7 @@ func NewE2ETestSuite(cfg network.Config) *E2ETestSuite {
 // SetupSuite initializes machine E2ETestSuite
 func (s *E2ETestSuite) SetupSuite() {
 	s.T().Log("setting up e2e test suite")
-
+	lib.GetConfig()
```

Comment:
there is already a `lib.GetConfig()` in `network.New(s.T())`. Could you investigate why this is not sufficient and maybe move it up a bit?

Reply:
true! I mixed upt the GetConfig context. I reintegrated the old init again